src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,42 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $font-color;
+    font-weight: $font-weight-medium;
+    font-size: 12px;
+    line-height: 14px;
+    height: 60px;
+    display: flex;
+    align-items: center;
+
+    &:hover {
+      color: $blue-accent;
+    }
+  }
+}
+
+.is-active {
```

Comment:
The '.is-active' class should be a modifier of a specific element rather than a standalone class. This way, it will be more consistent with the BEM methodology.